loopchain/channel/channel_inner_service.py

```diff
@@ -620,6 +620,10 @@ def get_tx_info(self, tx_hash):
                 logging.error(f"get_tx_info error : tx_hash({tx_hash}) not found error({e})")
                 response_code = message_code.Response.fail_invalid_key_error
                 return response_code, None
+            except PrunedHashDataError as e:
+                logging.debug(f"Prune Hash Data ({tx_hash}), {e}")
+                response_code = message_code.Response.pruned_hash_data
+                return response_code, None
```

Comment:
please, update log message to easy to understand.
```suggestion
                logging.warning(f"tx_hash({tx_hash}), {e!r}")
                response_code = message_code.Response.pruned_hash_data
                return response_code, None
```